rhminer-cpu/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+lscpu
+CORES=$(lscpu | head -5 | grep "CPU(s):" | awk '{print $2}')
```

Comment:
You're using `head` command to avoid the `NUMA node0 CPU(s):` line in the output, right? 
But `grep` can be instructed to find lines that start with given pattern:

```suggestion
CORES=$(lscpu | grep "^CPU(s):" | awk '{print $2}')
```